libs/shared/src/data/enums.ts

```diff
@@ -115,3 +115,8 @@ export enum CustomDeliveryType {
   Monday = 'monday',
   Saturday = 'saturday',
 }
+
+export enum LimitedPackageTypeName {
```

Comment:
I suppose there is an enum with all the package type names already? If so I would use it to fill the PACKAGE_TYPE_DEFAULT constant with the two relevant names, rather than duplicate package type names in this enum.

Reply:
Thank you for the suggestion. Yes, there is an enum with all the package type names. I will update the PACKAGE_TYPE_DEFAULT constant to use the relevant names from the existing enum to avoid duplication.